internal/check/check.go

```diff
@@ -423,3 +425,11 @@ func NotPresent[T any, QT ygnmi.SingletonQuery[T]](query QT) Validator {
 		return nil
 	})
 }
+
+// UnorderedEqual function is used to compare slices in un ordered way.
+func UnorderedEqual[T any, QT ygnmi.SingletonQuery[T]](query QT, want T) Validator {
+	return Predicate(query, fmt.Sprintf("want %#v", want), func(got T) bool {
+		// Sort slices to compare them in unorderd way.
+		return cmp.Equal(got, want, cmpopts.SortSlices(func(a, b string) bool { return a < b }))
```

Comment:
Can we make `func(a, b string) bool { return a < b }` part of the argument to be supplied by caller?

Reply:
updated, please check